src/components/common/YourCompany/BusinessContactInfo.vue

```diff
@@ -65,6 +66,9 @@ export default class BusinessContactInfo extends Mixins(CommonMixin) {
 
   /** Update Contact info. */
   private async onContactInfoChange (contactInfo: ContactPointIF): Promise<void> {
+    // do nothing if contact info was not changed
+    if (isEqual(contactInfo, this.getBusinessContact)) return
```

Comment:
Ideally, the shared component would not emit a change when the user clicks Cancel, but I think this is an acceptable work-around for now.

(This code change is not needed to fix a bug; it's just good practice.)